scipy/sparse/sparsetools/coo.h

```diff
@@ -112,6 +112,53 @@ void coo_todense(const I n_row,
 }
 
 
+/*
+ * Input Arguments:
+ *   I  shape[num_dims]  - shape of A
+ *   npy_int64  nnz      - number of nonzeros in A
+ *   npy_int64  num_dims - number of dimensions of A
+ *   I  Aijk[nnz(A)]     - coords for nonzeros in A
+ *   T  Ax[nnz(A)]       - nonzeros in A
+ *   T  Bx[]             - dense array
+ *
+ */
+template <class I, class T>
```

Comment:
This is consistent with the other sparsetools code, but I'm just noticing it now because we're having persistent issues with using `I` as variable name (see for example gh-21376). Let's rename them all in a separate PR - we can touch on this today (not blocking for this PR).